docs/manual.xml.head

```diff
@@ -4560,8 +4560,11 @@ bind index gg first-entry
       </para>
       <cmdsynopsis>
         <command>folder-hook</command>
+        <arg choice="opt">
+          <replaceable class="parameter">-noregex</replaceable>
+        </arg>
         <arg choice="plain">
-          <replaceable class="parameter">regex</replaceable>
+          <replaceable class="parameter">pattern</replaceable>
```

Comment:
Hmm... I'm not sure about this.
In the docs 'pattern' generally refers to a NeoMutt pattern, e.g. `~f addr`

Perhaps: `string_to_match`, `match_string`
Hmm... I'm not convinced by those.